api/src/main/java/org/pmiops/workbench/auth/ProfileService.java

```diff
@@ -58,6 +58,14 @@ public Profile getProfile(User user) {
     }
 
     boolean enabledInFireCloud = fireCloudService.isRequesterEnabledInFirecloud();
+    Profile profile = getTrimmedProfile(user);
+    profile.setEnabledInFireCloud(enabledInFireCloud);
```

Comment:
This has been a big source of slowness in general e.g. on /me. I don't understand why we need this on the profile (let alone why it's not stored) - can this just be removed entirely? There don't seem to be any useful references to it.